docs/install/remote.md

```diff
@@ -70,7 +70,7 @@ Set environment variables to define your MLRun configuration. As a minimum requi
     MLRUN_DBPATH=<URL endpoint of the MLRun APIs service endpoint; e.g., "https://mlrun-api.default-tenant.app.mycluster.iguazio.com">
     ```
 
-2. If the remote service is on an instance of the Iguazio MLOps Platform (**"the platform"**), set the following environment variables as well:
+2. If the remote service is on an instance of the Iguazio MLOps Platform (**"not relevant for MLRun CE"**), set the following environment variables as well:
```

Comment:
2. If the remote service is on an instance of the Iguazio MLOps Platform (**not relevant for MLRun CE**), set the following environment variables as well:

Reply:
Fixed